src/BuildingBlocks/BuildingBlocks.csproj

```diff
@@ -7,19 +7,24 @@
     </PropertyGroup>
 
     <ItemGroup>
+      <PackageReference Include="BuildingBlocks.Common" Version="1.0.0.65" />
```

Comment:
There should have been no need for additional packages. At least Microsoft.Tpl.Dataflow is very unnecessary